### ISSUE TITLE
[São Paulo] Data Scientist na @RecargaPay

### ISSUE THREAD

**Issue by leticia-morito** [datascience-br/vagas#27]
## Descrição da vaga

We are looking for a Senior Data Scientist to analyze large amounts of raw information to find patterns that will help improve our company. We will rely on you to build data products to extract valuable business insights. In this role, you should be highly analytical with a knack for analysis, math and statistics. Critical thinking and problem-solving skills are essential for interpreting data. We also want to see a passion for machine-learning and research. Your goal will be to help our company analyze trends to make better decisions.

Develop and improve Credit and Collection models using Machine Learning;
Build new relevant features to Credit and Collection models;
Identify opportunities with studies and analyzes with market information;
Solve real problems in business of Credit and Collection;
Conduct studies and backtesting with services and providers of credit and collections solutions.

## Local

São Paulo - Faria Lima

## Benefícios

Medical and dental assistance - You don't have to worry about co-participation fees. Here our RecargaPayers have access to excellent hospitals throughout Brazil, without worrying about discounts at the end of the month.
Life insurance - Because we know that your life is worth a lot <3
Transportation Voucher or Parking - We discount only 1 Real from it. Cool, right?
Flexible Meal voucher - What does it mean? You can choose how you want to spend your $$ in Lunch, Market, Pharmacy, Courses, Mobility, Education ... yes, lots of possibilities.
TotalPass - Healthy mind, healthy body.
Money in your RecargaPay Wallet monthly - To pay for that one bill and still have cashback. And specially to tell us how we can improve even more.
Spanish lessons - Sí, acá trabajamos aprendiendo.
Up to 70% discount on English courses - Why not be trilingual?
Discount in Universities for Undergraduate, Graduate and MBA courses - Knowledge is never too much, is it?
No Dress-Code - Here you can be YOU!

## Requisitos

**Obrigatórios:**
Bachelor degree or higher degree in Statistics, Mathematics, Computer Science, Engineering or related courses;
Proven experience as a Data Scientist or Data Analyst;
Understanding of machine-learning and operations research
Experience using statistical computer languages, for example Python (Numpy, Pandas, Scikit-learn, Keras, etc), SQL, SPSS e R;
Experience with Machine Learning Models: GLM / Regression, Boosting, XGBoost, Random Forest, LightGBM, Text Mining, etc.;
Analytical mind and business acumen;
Strong math skills (e.g. statistics, algebra);
Strong analytical profile, excellent organization and interpersonal relationships.

**Diferenciais:**
Experience in the areas of credit and collection;
Knowledge of the main credit and collection products in the financial market;
Knowledge of projects with agile methodologies;
Experience with Artificial Intelligence models;
Experience with data computing tools: Hadoop, Spark, MySQL, etc.
English or Spanish is a plus.

## Contratação

CLT

## Nossa empresa

RecargaPay is the leading mobile payments App in Brazil. A fintech of day-to-day transactions that meets the needs of bankers and unbanked people across the country, from consumers to traders.
We believe that everyone should have access to the benefits of mobile payments, so we are committed to democratizing these transactions, allowing their access to the whole of society.
We are looking for passionate people willing to embark on this adventure!

## Como se candidatar

If you are interested, send your CV to **leticia.morito@recargapay.com** or apply through our careers page: **https://recargapay.com.br/carreiras**

## Labels

- Alocado
- CLT
- Pleno
- Sênior
